README.md

```diff
@@ -32,26 +33,36 @@ You can get any updates or progress on my [Discord](https://discord.gg/Py2vSwg3B
 
 ### Default Configuration
 
-``max_games: 3``
+*&max_games:** ``3``
 
-``world_name: amongus``
+**world_name:** ``amongus``
 
 ### Commands
 
 - ``/au`` - Opens the Main UI for the plugin.
-- ``/aum`` - Game setup UI.
+- ``/aum`` - Opens the Game setup UI.
+
+### Permissions
+
+You need ``amongus.command.manage`` to use Game Setuo Menu.
+
+### Dependencies Plugins
+
+- muqsit's InvMenu Virion [Download](https://github.com/Muqsit/InvMenu)
```

Comment:
Muqsit's InvMenu is packed with phar(using poggit), user don't needed to download them.